audiobookshelf/docker-compose.yml

```diff
@@ -5,9 +5,12 @@ services:
     environment:
       APP_HOST: audiobookshelf_web_1
       APP_PORT: 80
+      PROXY_AUTH_WHITELIST: "/"
+      PROXY_AUTH_BLACKLIST: "/login,/api,/config"
```

Comment:
@ceramicwhite I think some adjustment is still needed here. I've been testing with the iOS app and I am getting `Unknown Error` when trying to login. Without digging deeper - perhaps the app (like the webapp) is using `/login` for auth too? But here it's marked a blacklisted (ie protected by the App Proxy).

Reply:
Hmm strange, I just tried it again with a fresh install worked fine. On my main ABS server I tell friends to click login twice in the mobile app because first time it usually throws an error, fails handshake or something. Maybe that's what happened?

Reply:
This is what I'm seeing:
![IMG_1772](https://user-images.githubusercontent.com/1112574/210672669-f2b1e567-25a8-49ab-8f9d-03404433e1ef.PNG)

If I remove `/login` from the App Proxy blacklist, then it works - I'm happy with that as a solution because audiobookshelf already has it's own auth. system. But I do see that on first install, that `/login` is used to set the inital `root` user password...

Reply:
sounds good!